src/views/Order.vue

```diff
@@ -81,15 +85,15 @@
                 <ion-list-header>{{ $t("Shopify IDs") }}</ion-list-header>
                 <ion-item>
                   <ion-label> {{ $t("Order Number") }} </ion-label>
-                  <p slot="end">14 Jan 2021</p>
+                  <p slot="end">{{ $filters.getIdentification(order.identifications, orderNo) ? $filters.getIdentification(order.identifications, orderNo) : "-" }}</p>
```

Comment:
We could process it while setting

Reply:
Processed the order identification values when getting the response.